### PR TITLE
Music page (front-end)

### PR DESCRIPTION
**Pull request by aishahmed**

### CODE REVIEW

File: client/src/components/pages/music.tsx [hunk 1]
```diff
@@ -1,11 +1,168 @@
...
+      style={{
+        display: "flex",
+        flexDirection: "column",
+        justifyContent: "flex-start",
+        alignItems: "center",
+        height: "100vh",
+        fontFamily: "Arial, sans-serif",
+        textAlign: "center",
+        padding: "20px",
+        color: "white",
```
Comment (Shuzhengz) [inline client/src/components/pages/music.tsx:59]: prob create a separate `music.css` for all the css stuff

File: client/src/components/pages/music.tsx [hunk 2]
```diff
@@ -1,11 +1,76 @@
...
-
-const Music = () => {
-    return (
-        <>
-            <h1>Music</h1>
-        </>
-    );
+import { useState } from "react";
+import useSound from "use-sound"; // for handling the sound
+import { AiFillPlayCircle, AiFillPauseCircle } from "react-icons/ai"; // icons for play and pause
```
Comment (Shuzhengz) [inline client/src/components/pages/music.tsx:3]: has error:
> Module not found: Error: Can't resolve 'react-icons/ai' in '/home/runner/work/SleepBuddies/SleepBuddies/client/src/components/pages'

**Review by Shuzhengz — CHANGES_REQUESTED** [review]

File: client/src/components/pages/music.tsx (hunk 2 above)
Comment (aishahmed) [reply client/src/components/pages/music.tsx:3]: Latest pull request should resolve this issue.

**Comment by Shuzhengz** [comment]
ok do `git pull origin music-branch` and then in `client` do `npm install` and that should solve the dependency issue

**Review by Shuzhengz — APPROVED** [review]
Nice

**Comment by Shuzhengz** [comment]
fyi the tester outputted this

```
Failed to compile.

[eslint] 
src/components/pages/music.tsx
  Line 57:6:  React Hook useEffect has a missing dependency: 'isPlaying'. Either include it or remove the dependency array  react-hooks/exhaustive-deps
```

File: client/src/Playlist.test.tsx
Comment (Shuzhengz) [inline client/src/Playlist.test.tsx:1]: This is failing the test because it's empty

**Review by jut008 — APPROVED** [review]
lgtm

**Pull request merged**